masterlist.yaml

```
@@ -5585,6 +5590,50 @@ plugins:
   - name: 'Recastable Summons - Mighty Magicka Version.esp'
     msg: [ *patchOutdated ]
 
+  - name: 'SupremeMagicka.esp'
+    url:
+      - link: 'https://www.nexusmods.com/oblivion/mods/10791/'
+        name: 'Supreme Magicka'
+      - link: 'https://www.nexusmods.com/oblivion/mods/12466/'
+        name: 'Supreme Magicka Update'
+    after:
+      - name: 'bgMagicEV.esp'
+        display: 'Less Annoying Magic Experience (L.A.M.E)'
+        condition: 'file("bgMagicEV_Conjuration_for_Supreme_Magicka.esp")'
+      - 'lumi_enchantments.esp'
+      - 'lumi_enchantments_subtle.esp'
+    # doNotClean
+    msg: [ *obsolete ]
```

You could link the updated version in the main file.
```suggestion
    msg:
      - <<: *obsolete
        subs: [ '[Supreme Magicka Update](https://www.nexusmods.com/oblivion/mods/12466/)' ]
```